app/lib/Plugins/SearchEngine/SqlSearch.php

```diff
@@ -91,12 +92,19 @@ public function __construct($po_db=null) {
 		
 		$this->opo_tep = new TimeExpressionParser();
 		$this->opo_tep->setLanguage($g_ui_locale);
-		
-		$this->opo_stemmer = new SnoballStemmer();
+
+		// TODO: Update to use an non-English stemmer
+        $vs_stemmer = $this->opo_search_config->get('search_sql_search_stemmer');
+        if (!$vs_stemmer){
+            $vs_stemmer = self::SNOBALL_STEMMER;
+        }
+        $vo_stemmer_factory = StemmerFactory::get_instance();
+		$this->opo_stemmer = $vo_stemmer_factory->create($vs_stemmer);
 		$this->opb_do_stemming = (int)trim($this->opo_search_config->get('search_sql_search_do_stemming')) ? true : false;
 		
 		$this->initDbStatements();
-		
+
+		// TODO: ops_indexing_tokenizer_regex is not used in this class
```

Comment:
We can get rid of this. At one point we were tokenizing in some cases at this level, but that's not coming back.